libs/docs/src/docs/no-version/quickstart-odo.md

```diff
@@ -0,0 +1,219 @@
+---
+title: "Quick start with odo v3"
+description: "Quick start with odo v3"
+---
+
+This guide will run through creating a simple hello world devfile project using `odo` version 3. The purpose of this guide is to provide a practical introduction to those just starting out with devfiles.
+
+## Prerequisites
+
+- `odo` v3
+- `kubectl` or `oc`
+- Docker 17.05 or higher
```

Comment:
This guide uses only `odo dev` and devfile being created as part of this doesn't have `image` component so Docker is not required here.